squad/frontend/comparison.py

```diff
@@ -35,3 +35,36 @@ def compare_test(request):
 
     context = {}
     return render(request, 'squad/compare.jinja2', context)
+
+
+def compare_builds(request):
+    project_slug = request.GET.get('project')
+    user = request.user
+    projects = Project.objects.accessible_to(user).prefetch_related('group')
```

Comment:
this can (and eventually will) be too many projects.

maybe we can start with this, and come up later with an UI element to select a project by typing (parts of) its name.

Reply:
I've added a select2 box for project and builds. Now by clicking on it, a few options will show up automagically, then options will be filtered out as you type